### PR TITLE
implement intrinsic load and store 64 bit values

### LINKED ISSUE #51: Supporting Intrinsic `LOAD` and `STORE` 64 bit values
I was interested in contributing to this, so I figured I should ask how you're thinking this should be implemented. 
Mainly, should the size of the value to be read be an input (like 8, 16, 32, 64 bit) OR should it just have intrinsic support for 64 bit values and then have standard library macros provide support for other sizes?

If its only 64 bit then it can probably just be done with

```asm
pop rax
xor rbx, rbx
mov rbx, [rax] ;; Instead of mov bl, [rax]
push rbx
```

### PR DESCRIPTION
**Pull request by zrthxn**
Fixes #51

### CODE REVIEW

File: porth.py [hunk 1]
```diff
@@ -249,6 +251,16 @@ def simulate_little_endian_linux(program: Program):
...
                 ip += 1
+            elif op.operand == Intrinsic.LOAD64:
+                addr = stack.pop()
+                byte = mem[addr]
+                stack.append(byte)
+                ip += 1
+            elif op.operand == Intrinsic.STORE64:
+                store_value = stack.pop()
+                store_addr = stack.pop()
+                mem[store_addr] = store_value & 0xFFFFFFFF
```
Comment (rexim) [inline porth.py:262]: `0xFFFFFFFF` is a 32 bit mask, not 64. 64 would be `0xFFFFFFFFFFFFFFFF` and I'm not even sure if it's even needed.

**Review by rexim — CHANGES_REQUESTED** [review]
- Add some tests for load64 and store64 intrinsics
- Remove load64 and store64 macros from std.porth
- Fix the solutions in ./euler/ that use load64 and store64 macros
- Add documentation for .64 and ,64 to README

File: porth.py (hunk 1 above)
Comment (zrthxn) [reply porth.py:262]: Oh I seem to have miscounted. Will fix.

**Comment by zrthxn** [comment]
The simulation runs fail because the memory is a bytearray. The following can be done to replace the existing python 
```python
            elif op.operand == Intrinsic.LOAD64:
                addr = stack.pop()
                for offset in range(0,7):
                    byte = mem[addr + offset]
                    stack.append(byte)
                ip += 1
            elif op.operand == Intrinsic.STORE64:
                store_value = stack.pop()
                store_addr = stack.pop()
                for offset in range(0,7):
                    mem[store_addr + offset] = store_value & 0xFF
                    store_value >> 8
                ip += 1
```

**Comment by rexim** [comment]
@zrthxn you may wanna look into https://docs.python.org/3/library/stdtypes.html#int.to_bytes

**Comment by zrthxn** [comment]
@rexim Thanks! Didn't think of that. It works now

**Comment by rexim** [comment]
@zrthxn looks good to me! :+1: Thank you for your contribution!

**Pull request merged**